tests/test_job.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from pybuildkite.jobs import Jobs
+from pybuildkite.client import Client
+from requests.exceptions import HTTPError
+
+class TestJob:
+
+    client = Client()
```

Comment:
Does it make sense to use some setup/teardown functions here for these variables?

Reply:
Yeah you are right I think I should use setup/teardown functions let me fix that.